sound/soc/sof/Kconfig

```diff
@@ -55,6 +55,13 @@ config SND_SOC_SOF_DEBUG_PROBES
 	  Say Y if you want to enable probes.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_CLIENT
+	bool
```

Comment:
is there a reason not to make this a tristate? This will force the auxiliary bus to be built-in, it's not really necessary, is it?

Reply:
I use it as a flag to include the sof-client support or not.